app.js

```diff
@@ -264,11 +264,14 @@ function updateStreamsList(cropsDir) {
       if (array.length > 0) {
         console.log(array);
         console.log("lowest stream: " + array[0].name);
-        currentStream = streamToObject(array[0]);
-        for (let idx in array) {
-          newAllStreams.push(streamToObject(array[idx]));
+        // don't switch inside 10 players; don't stay on stream post-game
```

Comment:
The app should always expose the closest value to truth as possible. The client-side js is the place to implement 'dont switch when....' logic.

For the 'alive == 1' case, we can handle that in the /process_pubg route on the ocr service.